modules/files/files.tsp

```diff
@@ -8,6 +8,6 @@ model FilesModule {
      */
     type: "files";
 
-    /** List of files / folders to copy. Format is: `sourcefile.txt: /destination/file.txt`  */
+    /** List of files / folders to copy. */
     files: Array<Record<string>>;
```

Comment:
This is incorrect. There is an OR-union in TypeSpec, but I can't test that now. Either we merge with incomplete or untested changes or do this in a couple days.

Reply:
Well, it can wait I guess

Reply:
Is there a need to support old format in typespec though?

Reply:
I guess it should be something like this
```suggestion
    files: Array<Record<string>> | Array<{
        source: string;
        destination: string;
    }>;
```
That seems to work on the playground. https://typespec.io/playground

Problem solved, I guess.

Reply:
Oh, this playground thing is cool